ibm_catalog.json

```diff
@@ -128,7 +169,129 @@
+                "on_by_default": true,
+                "default_flavor": "standard",
+                "input_mapping": [
+                  {
+                    "reference_version": true,
+                    "dependency_input": "resource_group_name",
+                    "version_input": "existing_resource_group_name"
+                  },
+                  {
+                    "value": true,
+                    "dependency_input": "use_existing_resource_group"
+                  },
+                  {
+                    "reference_version": true,
+                    "dependency_input": "region",
+                    "version_input": "region"
+                  },
+                  {
+                    "reference_version": true,
+                    "dependency_input": "prefix",
+                    "version_input": "prefix"
+                  }
+                ],
+                "ignore_auto_referencing": [
+                  "*"
+                ]
+              },
+              {
+                "name": "deploy-arch-ibm-account-infra-base",
+                "id": "63641cec-6093-4b4f-b7b0-98d2f4185cd6-global",
+                "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+                "optional": true,
+                "on_by_default": true,
+                "default_flavor": "standard",
+                "version": "^v1.18.2",
+                "input_mapping": [
+                  {
+                    "reference_version": true,
+                    "dependency_input": "prefix",
+                    "version_input": "prefix"
+                  },
+                  {
+                    "dependency_input": "skip_iam_account_settings",
+                    "value": true
+                  },
+                  {
+                    "dependency_output": "workload_resource_group_name",
+                    "version_input": "existing_resource_group_name"
+                  }
+                ],
+                "ignore_auto_referencing": [
+                  "*"
+                ]
+              }
+            ],
+            "dependency_version_2": true,
+            "configuration": [
+              {
+                "key": "ibmcloud_api_key",
+                "required": true
+              },
+              {
+                "key": "provider_visibility",
+                "options": [
+                  {
+                    "displayname": "private",
+                    "value": "private"
+                  },
+                  {
+                    "displayname": "public",
+                    "value": "public"
+                  },
+                  {
+                    "displayname": "public-and-private",
+                    "value": "public-and-private"
+                  }
+                ]
+              },
+              {
+                "key": "existing_resource_group_name",
+                "required": true
+              },
+              {
+                "key": "region",
```

Comment:
This should be a region picker. We need to decide which one to use based on where DB2 is available. There is the region picker and the VPC region picker, and if neither of those is appropriate, we can define our own custom list of regions.